contrib/packs/actions/install.meta.yaml

```diff
@@ -21,5 +21,5 @@
       default: "master"
     register:
       type: "string"
-      default: "actions"
+      default: "actions,aliases,sensors"
```

Comment:
> I tried with array first, but it's converted into string. See: #1231 (comment)

Most likely due to the specific jinja template being used. Have not tried it out myself so can't say what works and what does not.

> Also, as I understand array type won't be preserved here in install workflow

It will if load.yaml say it expect input as an array. The output type is not preserved since we lack type information. However, input to an action is cast, as much as possible, into the correct types.

I will agree that all this is very confusing and how the translations happens and types are preserved/not preserved during transitions is quite unclear.